doc/nrf/ug_thread_commissioning.rst

```diff
@@ -208,8 +206,3 @@ See the following pages in the `OpenThread CLI Reference`_ on GitHub for an over
 
 * `Commissioner CLI commands`_
 * `Joiner CLI commands`_
-
-----
-
-Copyright disclaimer
```

Comment:
I started to wonder, Is it acceptable to remove this? Shouldn't there be a copyright disclaimer on every page?

Reply:
I'll revert this change for now, as it's a bit beyond the scope of this PR.